src/components/PaymentStripe/cardSection.js

```diff
@@ -0,0 +1,136 @@
+import React, { useMemo, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { CardElement, useElements, useStripe } from '@stripe/react-stripe-js';
+import { useTheme, useUniqueIdentifier } from '../../libs/hooks';
+import RenderIf from '../RenderIf';
+import Label from '../Input/label';
+import HelpText from '../Input/styled/helpText';
+import ErrorText from '../Input/styled/errorText';
+import StyledContainer from './styled/container';
+
+const CardSection = props => {
+    const {
+        label,
+        hideLabel,
+        bottomHelpText,
+        error,
+        className,
+        style,
+        hidePostalCode,
+        hideIcon,
+        iconStyle,
+        disabled,
+        postalCode,
+        onChange,
+        onReady,
+        onFocus,
+        onBlur,
+    } = props;
+    const stripe = useStripe();
+    const elements = useElements();
+    const cardElementId = useUniqueIdentifier('card-element');
+    const theme = useTheme().rainbow;
+    const cardElementOptions = useMemo(
+        () => ({
+            style: {
+                base: {
+                    iconColor: theme.palette.text.main,
+                    fontFamily: '"Lato", Arial, sans-serif',
+                    backgroundColor: theme.palette.background.main,
+                    color: theme.palette.text.main,
+                    fontSize: '1.1rem',
+                    '::placeholder': {
+                        color: theme.palette.text.header,
+                        fontVariant: 300,
+                    },
+                },
+            },
+            hidePostalCode,
+            hideIcon,
```

Comment:
let's remove the logic for removing the postalCode and hide the card icon if we have the use case in future we add it